### ISSUE TITLE
TypeError `_internal_priceScale is not a function` while getting series price scale

### ISSUE THREAD

**Issue by timocov** [tradingview/lightweight-charts#536]
**Lightweight Charts Version:** 3.1.3

**Steps/code to reproduce:**

```js
const chart = LightweightCharts.createChart(document.body);
const series = chart.addAreaSeries();
const priceScale = series.priceScale();
```

**Actual behavior:**

Error `TypeError: this._private__priceScaleApiProvider._internal_priceScale is not a function`

**Expected behavior:**

A price scale api is returned.

**Comment by timocov** [comment]
It seems that the fix doesn't help :(

**Comment by timocov** [comment]
The issue was due outdated version of `ts-transformer-properties-rename` (I've checked the fix on master, this is way I didn't handle the issue on released tag).

**Comment by timocov** [comment]
I just re-published a proper fix in 3.1.5. Sorry for inconvenience.
